shell/client/styles/_topbar.scss

```diff
@@ -75,8 +75,7 @@ body>.topbar {
   background-color: $topbar-background-color;
   color: #ccc;
 
-  // Using 0 rather than CSS default ("auto") allows introjs to highlight elements in the topbar.
-  z-index: 0;
+  z-index: 1;
```

Comment:
Correct.  It works at `z-index: 0` because the first ancestor with a stacking context of the other element that needs to stack on top of the navbar is also at the same depth in the DOM and `z-index: 0`, so it's order is computed in the same stacking context (and stacks on top because it's further down in the DOM), until you set `z-index: 1`, in which case it doesn't any more.